### ISSUE TITLE
Duckstation's Guncon reticle is inaccurate in Judge Dredd (USA)

### ISSUE THREAD

**Issue by givedreddasequel2021** [stenzek/duckstation#2734]
I'll ask again. Why is shooting with the guncon in Judge Dredd so inaccurate in Duckstation? The white reticle is where I'm aiming, the orange one is where the shot is actually firing. This is with all settings default. Aiming with the controller is fine. The issue does not happen in Beetle PSX or on real hardware.

![1637137976844](https://user-images.githubusercontent.com/94535380/142196050-22ee68b9-a7cb-4e0c-ac9b-b2c49a29e4f3.png)

**Comment by bslenul** [comment]
Yeah the crosshair seems to be misaligned, just recalibrate the gun in the in-game menu, it will ask to shoot white squares, aim below it and a bit to the right, if it's still off do it again and adjust.

Fullscreen on my 1080p monitor calibrating like this seems pretty good:

![image](https://user-images.githubusercontent.com/33353403/142199560-7bb583de-6441-40fd-800b-e00d48a32f9c.png)

![image](https://user-images.githubusercontent.com/33353403/142199783-7799231d-03e1-40df-b924-55a9641b1bf8.png)
